recipes/cudnn-frontend/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "python-cudnn-frontend" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA/cudnn-frontend/archive/v{{ version }}.tar.gz
+  sha256: 8fa8ef57b7b42a889fcb97ff90c2b0fc171d80468dcf7c7935f0abd9d9757f1b
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  script_env:
+    - CUDNN_FRONTEND_LOG_INFO=1
+  number: 0
+  skip: true           # [cuda_compiler_version in (undefined, "None")]
+  skip: true           # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - {{ stdlib("c") }}
+    - cmake >=3.17
+    - cudnn
```

Comment:
This should be in host and then you probably don't need to explicitly mention it in run anymore.

Reply:
Ok, done! Thanks again for checking :smiley: